manifest.php

```diff
@@ -60,13 +61,58 @@
         ],
     ],
     'update' => 'taoItems_scripts_update_Updater',
-    'managementRole' => 'http://www.tao.lu/Ontologies/TAOItem.rdf#ItemsManagerRole',
+    'managementRole' => TaoItemsRoles::ITEM_MANAGER,
     'acl' => [
-        ['grant', 'http://www.tao.lu/Ontologies/TAOItem.rdf#ItemsManagerRole', ['ext' => 'taoItems']],
-        ['grant', 'http://www.tao.lu/Ontologies/TAOItem.rdf#AbstractItemAuthor', 'taoItems_actions_ItemContent'],
-        ['grant', 'http://www.tao.lu/Ontologies/TAO.rdf#DeliveryRole', ['ext' => 'taoItems', 'mod' => 'ItemRunner']],
-        ['grant', TaoRoles::REST_PUBLISHER, ['ext' => 'taoItems', 'mod' => 'RestItems']],
-        ['grant', TaoRoles::REST_PUBLISHER, ['ext' => 'taoItems', 'mod' => 'RestFormItem']],
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_MANAGER,
+            ['ext' => 'taoItems']
+        ],
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_AUTHOR_ABSTRACT,
+            'taoItems_actions_ItemContent'
+        ],
+        [
+            AccessRule::GRANT,
+            TaoRoles::DELIVERY,
+            ['ext' => 'taoItems', 'mod' => 'ItemRunner']
+        ],
+        [
+            AccessRule::GRANT,
+            TaoRoles::REST_PUBLISHER,
+            ['ext' => 'taoItems', 'mod' => 'RestItems']
+        ],
+        [
+            AccessRule::GRANT,
+            TaoRoles::REST_PUBLISHER,
+            ['ext' => 'taoItems', 'mod' => 'RestFormItem']
+        ],
+        [
```

Comment:
We will probably need to create multiple files for roles in the future, cause as more we segregate, bigger this file becomes.

Just a note to think about, no action needed :)